src/tt_citites_skylines/guide_grid.rb

```diff
@@ -6,32 +6,37 @@
 #-------------------------------------------------------------------------------
 
 module TT::Plugins::CitiesSkylinesTools
-
+    
   SECTOR_SIZE = 8.m
   OBJECT_TYPE = "ObjectType".freeze
   TYPE_GUIDE_GRID = "GuideGrid".freeze
-
   GRID_CELLS_X = "GridCellsX".freeze
   GRID_CELLS_Y = "GridCellsY".freeze
+  GRID_CELLS_SUBDIVS = "GridCellsSubDivs".freeze
+  HEIGHT_GRID = "HeightGrid".freeze
+  FLOORS = "Floors".freeze
+  FIRST_FLOOR_HEIGHT = "FirstFloorHeight".freeze
+  FLOOR_HEIGHT = "FloorHeight".freeze
+  
+  
+  def self.create_guide_grid(cells_x = nil, cells_y = nil, cells_subdivs = nil, height_grid = nil, floors = nil, first_floor_height = nil , floor_height = nil, group = nil)
+    cells_x = Sketchup.read_default(PLUGIN_ID, GRID_CELLS_X, 4) unless cells_x
+    cells_y = Sketchup.read_default(PLUGIN_ID, GRID_CELLS_Y, 4) unless cells_y
+    cells_subdivs = Sketchup.read_default(PLUGIN_ID, GRID_CELLS_SUBDIVS, 3) unless cells_subdivs
+    # Height
+    height_grid = Sketchup.read_default(PLUGIN_ID, HEIGHT_GRID, "No") unless height_grid
```

Comment:
The `.UI.inputbox` doesn't have checkboxes I'm afraid. The UI is rather limited. Dropdown boxes are the closest you get.

For this I usually use a hash to map the native values (booleans or integers etc) to strings for the inputbox.